AntennaTracker/Tracker.h

```diff
@@ -267,17 +278,27 @@ class Tracker : public AP_HAL::HAL::Callbacks {
     void prepare_servos();
     void set_mode(enum ControlMode mode, mode_reason_t reason);
     bool should_log(uint32_t mask);
+    bool start_command_callback(const AP_Mission::Mission_Command& cmd) { return false; }
+    void exit_mission_callback() { return; }
+    bool verify_command_callback(const AP_Mission::Mission_Command& cmd) { return false; }
 
     // tracking.cpp
     void update_vehicle_pos_estimate();
     void update_tracker_position();
     void update_bearing_and_distance();
     void update_tracking(void);
     void tracking_update_position(const mavlink_global_position_int_t &msg);
+    void adsb_tracking_update_position(mavlink_adsb_vehicle_t adsb_vehicle);
     void tracking_update_pressure(const mavlink_scaled_pressure_t &msg);
     void tracking_manual_control(const mavlink_manual_control_t &msg);
     void update_armed_disarmed();
+    bool stationary = true; // are we using the start lat and log?
```

Comment:
this is just so we don't send velocity when using a fixed locations, i guess you could check if the current location is equal to the start lat and long, i was thinking that it may happen, although quite unlikely, that we are in the start location without using the start location parameters so would still want to send velocitys. Maybe this is too paranoid?

Reply:
I dunno, it just seemed unnecessary. If you really like it and think we'll expand on it later then it can stay as-is.